pkg/infra/iac2/cloud_resources_stub.go

```diff
@@ -0,0 +1,20 @@
+package iac2
+
+type (
+	DockerLambda struct {
+		ExecUnitName    string
+		CloudwatchGroup CloudwatchLogGroup
+	}
+
+	CloudwatchLogGroup struct {
+		// fields...
+	}
+)
+
+func (cwg CloudwatchLogGroup) VariableName() string {
+	return "cloudwatchLogGroup"
+}
+
+func (dl DockerLambda) VariableName() string {
+	return "lambda_" + nonIdentifierChars.ReplaceAllString(dl.ExecUnitName, "_")
```

Comment:
Hm, if there's any characters that aren't `_`, this could create name conflicts, right? One alternative to consider, and this is a little clever, is to use the names as-is in a `Map` (or object properties) instead of as standalone variables. We could restrict the IDs for now to just be `[a-zA-Z_]+` for now and not need the replacement too.

Reply:
Yeah, I should have added a comment — this is very much just stubbed out for "good enough for a demo on Friday". Sorry I forgot to document that!

I think vars are a bit nicer for the user than a map, because they can use their IDE to jump around between usage and declaration. But what I'd like eventually is for (a) the name to be automatically generated from the resource type and id, and (b) if there are any conflicts (due to `_`-type smushing or anything else), have the `templatesCompiler` realize that and just assign each a unique number suffix:

```
id="foo/bar"    => const lambdaFooBar_0
id="foo-bar"    => const lambdaFooBar_1
```

Thoughts on that?

`lambdaFoo_Bar_1`